.github/workflows/codeql.yml

```
@@ -62,6 +62,12 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
+        with:
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)